src/server/db/seed/reviewSeeder.ts

```diff
@@ -0,0 +1,56 @@
+import { faker } from "@faker-js/faker";
+import { reviews } from "../schema";
+import { USERS } from "./userSeeder";
+import { UserPartial, type Seeder } from ".";
```

Comment:
```suggestion
import { UserPartial, type Seeder } from "./helpers";
```

this should be from helpers right?

Reply:
yuh, forgot I solved this locally but forgot to push it for joy